src/components/Blog/Newsletter.tsx

```diff
@@ -89,7 +88,7 @@ const BlogNewsletter = () => {
                 <label htmlFor="email">Email address</label>
 
                 <div className="form col">
-                  <input type_="text" name="email" onChange={handleInputChange} />
+                  <input type="text" name="email" onChange={handleInputChange} />
```

Comment:
Did you check this input still works after that change?

AFAIR this might have been `type_` on purpose, so it can be parsed and changed dynamically by 3rd party code.

Reply:
I've manually tested, works correctly 👍